src/pseudopeople/interface.py

```diff
@@ -30,12 +30,27 @@ def _generate_form(
         Noised form data
     """
     configuration_tree = get_configuration(configuration)
+    if isinstance(source, str):
+        source = Path(source)
     if isinstance(source, pd.DataFrame):
         data = source
+    elif isinstance(source, Path):
+        if source.suffix == ".hdf":
+            data = pd.read_hdf(source)
+            if not isinstance(data, pd.DataFrame):
```

Comment:
Should this be moved to after the parquet elif?

Reply:
Sure I can move it.